app/plugins/wamDataImporter/wamDataImporterPlugin.php

```diff
@@ -228,9 +228,10 @@ protected function _processListItemBundle(&$pa_table_content, $pa_translation_se
 			$va_attr_vals_with_parent['parent_id'] = $vn_parent_id;
 			$va_attr_vals_with_parent['is_enabled'] = 1;
 			$vs_item_idno = caGetOption('idno', $va_attr_vals_with_parent, is_array($vm_value) ? null : $vm_value);
+			$va_match_on = caGetOption('matchOn', $pa_translation_settings, array('label', 'idno'), array('castTo' => 'array'));
```

Comment:
This is only for identificationQualifier. CA defaults to `array('label','idno')` so I thought best to match that.